data_collection/api/tracking_api.yaml

```
@@ -434,14 +434,15 @@ paths:
               - [4] quantity: `integer` (optional, default to 1) Quantity of a product.
               - [5] brand: `string` (optional) a brand of the product
               - [6] variant: `string` (optional) a variant of the product
+              - [7] cutom product dimensions: `Map<integer, string>` a map of custom dimension values
 
             Legend: [index] name: `data type` (requisite) Desctiption. Restrictions.
 
             E-commerce cart must not contain more than 100 products.
           schema:
             type: string
             format: JSON
-          example: '%5B%5B%22craft-311%22%2C%22Unicorn%20Iron%20on%20Patch%22%2C%22Crafts%20%26%20Sewing%22%2C4.99%2C3%2C%22FairyTales%22%2C%223-color%20rainbow%22%5D%2C%5B%22craft-312%22%2C%22Unicorn%20Lamp%22%2C%5B%22Crafts%20%26%20Sewing%22%2C%20%22Lamps%22%5D%2C13.25%2C1%2C%22FairyTales%22%2C%22black-and-white%22%5D%5D'
+          example: '%5B%5B%22craft-311%22%2C%22Unicorn%20Iron%20on%20Patch%22%2C%22Crafts%20%26%20Sewing%22%2C4.99%2C3%2C%22FairyTales%22%2C%223-color%20rainbow%22%5D%2C%5B%22craft-312%22%2C%22Unicorn%20Lamp%22%2C%5B%22Crafts%20%26%20Sewing%22%2C%20%22Lamps%22%5D%2C13.25%2C1%2C%22FairyTales%22%2C%22black-and-white%22%2C%7B1%3A%22custom%20glitter%22%7D%5D%5D'
```

```suggestion
          example: '%5B%5B%22craft-311%22%2C%22Unicorn%20Iron%20on%20Patch%22%2C%22Crafts%20%26%20Sewing%22%2C4.99%2C3%2C%22FairyTales%22%2C%223-color%20rainbow%22%5D%2C%5B%22craft-312%22%2C%22Unicorn%20Lamp%22%2C%5B%22Crafts%20%26%20Sewing%22%2C%22Lamps%22%5D%2C13.25%2C1%2C%22FairyTales%22%2C%22black-and-white%22%2C%7B%221%22%3A%22custom%20glitter%22%7D%5D%5D'
```
Previous example could not be parsed by JSON

---

done